qiskit_experiments/data_processing/data_action.py

```diff
@@ -81,16 +102,42 @@ def __repr__(self):
 class TrainableDataAction(DataAction):
     """A base class for data actions that need training."""
 
+    def __init__(self, validate: bool = True):
+        """Create new node.
+
+        Args:
+            validate: If set to False the DataAction will not validate its input.
+        """
+        super().__init__(validate=validate)
+        self._parameters = self._default_parameters()
+
+    @classmethod
+    def _default_parameters(cls) -> Options:
+        """Set default node parameters."""
+        return Options()
+
+    @property
+    def parameters(self) -> Options:
+        """Returns the trained node parameters."""
```

Comment:
It is not guaranteed that these parameters are trained right? They might just be the default untrained parameters. Can you clarify.

Reply:
Exactly. This returns just a set of parameters that are available in current node.

Reply:
```suggestion
        """Return the parameters of the trainable node."""
```